src/ripple/basics/Buffer.h

```diff
@@ -34,14 +34,49 @@ namespace ripple {
 class Buffer
 {
 private:
-    std::unique_ptr<
-        std::uint8_t[]> p_;
+    std::unique_ptr<std::uint8_t[]> p_;
     std::size_t size_ = 0;
 
 public:
     Buffer() = default;
-    Buffer (Buffer const&) = delete;
-    Buffer& operator= (Buffer const&) = delete;
+
+    /** Create an uninitialized buffer with the given size. */
+    explicit
+    Buffer (std::size_t size)
+        : p_ (size ? new std::uint8_t[size] : nullptr)
```

Comment:
It's just how it was. I could add the `make_unique` but it doesn't really give us anything in this context.